picard/ui/infodialog.py

```diff
@@ -386,7 +390,14 @@ def sorttracknum(item):
                 except AttributeError:
                     return 0
 
-        lines = ["%s %s - %s (%s)" % item for item in sorted(tracklist, key=sorttracknum)]
-        info.append("<b>%s</b><br />%s" % (_('Tracklist:'),
-                    '<br />'.join([htmlescape(s).replace(' ', '&nbsp;') for s in lines])))
+        ndiscs = len(tracklists)
+        current_disc = -1
+        for discnumber in sorted(tracklists):
+            tracklist = tracklists[discnumber]
+            if ndiscs > 1 and current_disc != discnumber:
```

Comment:
Is `current_disc != discnumber` really a useful check here? You're iterating over the discnumbers, which are keys in a dictionary. I fail to see how you'd ever have the same discnumber in two separate iterations of this loop.

Reply:
You're right, totally useless, that was remaining bits from initial code. Thanks for catching it.